samples/cpp/benchmark_app/inputs_filling.cpp

```diff
@@ -208,15 +270,22 @@ ov::Tensor get_image_tensor(const std::vector<std::string>& files,
                             const std::pair<std::string, benchmark_app::InputInfo>& inputInfo,
                             std::string* filenames_used = nullptr) {
     auto type = inputInfo.second.type;
-    if (type == ov::element::f32) {
-        return create_tensor_from_image<float>(files,
+    if (type == ov::element::bf16) {
```

Comment:
Do we actually need to support bf16 input type?

Reply:
Removed, as agreed with Vladimir